pkg/operator/ceph/cluster/controller.go

```diff
@@ -173,6 +174,27 @@ func (c *ClusterController) StartWatch(namespace string, stopCh chan struct{}) {
 	} else {
 		logger.Infof("Disabling hotplug orchestration via %s", disableHotplugEnv)
 	}
+
+	// watch for "rook-ceph-operator-config" ConfigMap
+	configMapName := k8sutil.RookCephOperatorSetting
+	_, operatorConfigController := cache.NewInformer(cache.NewFilteredListWatchFromClient(c.context.Clientset.CoreV1().RESTClient(),
```

Comment:
How about adding this configmap watcher as a method in the `k8sutil` package? It could be called `StartOperatorSettingsWatch()`. For example, the arguments would be the configmap name and callback method.

Reply:
done.

Reply:
Do you still need to push this change? I don't see it.

Reply:
Yes, just pushed the changes.